src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+
+
+## What’s the impact of CMSs on the environment?
+
+This year we’ve sought to better understand the impact of CMSs on the environment. [The information and communications technology (ICT) industry accounts for 2% of global carbon emissions](https://www.nature.com/articles/d41586-018-06610-y), and data centers specifically account for 0.3% of global carbon emissions. This puts the ICT industry’s carbon footprint equivalent to the aviation industry’s emissions from fuel. We don’t have data on the role of CMSs here, but with our research showing 42% of websites use a CMS, it is clear CMSs play an important role in the efficiency of websites and their impact on the environment.
+
+Our research looked at the average CMS page weight in KB and mapped this to CO2 emissions using logic from [carbonapi](https://gitlab.com/wholegrain/carbon-api-2-0/-/blob/master/includes/carbonapi.php#L342). This generated the following results, split by desktop and mobile:
+
+{{ figure_markup(
+  image="cms-carbon-emissions-per-cms-page-view.png",
+  caption="Carbon Emissions per CMS page view.",
+  description="Bar chart showing the grams of CO2 emissions for CMS pages by percentile. At the 10th percentile it is 0.5 grams for desktop and 0.4 for mobile, at the 25th percentile it is 0.8 grams for both mobile and desktop, at the 50% it's 1.5 grams for both, at the 75th percentile 2.8 grams for desktop and 2.7 for mobile and at the 90th percentile it is 4.9 grams for desktop and 4.7 for mobile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=1375803209&format=interactive",
+  sheets_gid="1594044364",
+  sql_file="distribution_of_page_weight_requests_and_co2_grams_per_cms_web_page.sql"
+  )
+}}
+
+We found that the median CMS page load resulted in the transfer of 2.41 MB and thus the emission of 1.5g of CO2. This was the same for desktop and mobile. The most efficient percentile of CMS web pages result in the generation of at least one third less CO2, whilst the least efficient percentile of CMS web pages goes the other way: over one third less efficient than the median. The most efficient percentile of pages is approximately ten times more efficient than the least efficient percentile.
+
+CMSs power every type of website, so this discrepancy is not surprising. CMSs can, however, influence at the platform-level the efficiency of websites they create.
+
+Page weights are important here. The average desktop CMS web page loads 2.4 MB of video, image, script, font, CSS, and audio data. 10% of pages, however, load over 7 MB of this data. On mobile devices the average web page loads 0.1 MB fewer than on desktop, with at least this number being true across all percentiles:
```

Comment:
Enumerating all of these resource types makes the list sound exhaustive, but HTML isn't listed. Rather than trying to list all resource types maybe use an example like "2.4 MB of HTML, CSS, JavaScript, media, etc."

Reply:
cheers, changed!